_sources/SequenceMutation/ExtraExercises.rst

```diff
@@ -0,0 +1,44 @@
+..  Copyright (C)  Brad Miller, David Ranum, Jeffrey Elkner, Peter Wentworth, Allen B. Downey, Chris
+    Meyers, and Dario Mitchell.  Permission is granted to copy, distribute
+    and/or modify this document under the terms of the GNU Free Documentation
+    License, Version 1.3 or any later version published by the Free Software
+    Foundation; with Invariant Sections being Forward, Prefaces, and
+    Contributor List, no Front-Cover Texts, and no Back-Cover Texts.  A copy of
+    the license is included in the section entitled "GNU Free Documentation
+    License".
+
+.. qnum::
+   :prefix: seqmut-12
+   :start: 1
+
+Extra Exercises
+===============
+
+1. (You may work on on this one in class. Feel free to start thinking about it.) In Robert McCloskey's
```

Comment:
delete the You may work on this one in class, and provide an actex

Also, for all of these the instructions need to be embedded inside the actex or we won't capture the question text in the database.